tests/test_wafv2/test_wafv2.py

```diff
@@ -0,0 +1,45 @@
+from __future__ import unicode_literals
+import json
+import pytest
+
+import sure  # noqa\
+import boto3
+from botocore.exceptions import ClientError
+from moto import mock_wafv2
+from test_helper_functions import CREATE_WEB_ACL_BODY, LIST_WEB_ACL_BODY
+
+
+@mock_wafv2
+def test_create_web_acl():
+
+    conn = boto3.client("wafv2", region_name="us-east-1")
+    res = conn.create_web_acl(**CREATE_WEB_ACL_BODY("John", "REGIONAL"))
+    web_acl = res["Summary"]
+    assert web_acl.get("Name") == "John"
+    assert web_acl.get("ARN").startswith("arn:aws:wafv2:us-east-1:123456789012:regional/webacl/John/")
+    # Duplicate name - should raise error
+    with pytest.raises(ClientError):
+        conn.create_web_acl(**CREATE_WEB_ACL_BODY("John", "REGIONAL"))
+    res = conn.create_web_acl(**CREATE_WEB_ACL_BODY("Carl", "CLOUDFRONT"))
+    web_acl = res["Summary"]
+    assert web_acl.get("ARN").startswith("arn:aws:wafv2:global:123456789012:global/webacl/Carl/")
+
+
+@mock_wafv2
+def test_list_web_acl():
+
+    conn = boto3.client("wafv2", region_name="us-east-1")
+    conn.create_web_acl(**CREATE_WEB_ACL_BODY("Daphne", "REGIONAL"))
+    conn.create_web_acl(**CREATE_WEB_ACL_BODY("Penelope", "CLOUDFRONT"))
```

Comment:
Can we add an additional regional call here, and verify that it has persisted it as such:
```
conn.create_web_acl(**CREATE_WEB_ACL_BODY("Daphne", "REGIONAL"))
conn.create_web_acl(**CREATE_WEB_ACL_BODY("Penelope", "CLOUDFRONT"))
conn.create_web_acl(**CREATE_WEB_ACL_BODY("Sarah", "REGIONAL"))
```

Exactly in this order.
I'm expecting this to fail at the moment, because we're overriding the region in responses.py

Reply:
Added, although no error I think?

Reply:
Thanks for adding the test case. I was suprised it was passing, so I debugged it a bit - turns out the Response-object is recreated every time, so we can re-assign whatever we want without interfering with the next request.